web/looking-good/src/pages/user-profile-edit.js

```diff
@@ -46,117 +64,242 @@ const UserProfileEdit = () => {
   const [formValues, setFormValues] = useState({
     id: null,
     user_name: '',
+    profile_pic: null,
+    business_name: '',
+    specialty: '',
+    tip_url: '',
+    video_url: '',
+    blurb: '',
   })
   const router = useRouter()
   const { isLoggedIn, getUserProfile, updateUserProfile } = useAuth()
 
   useEffect(() => {
-    if (isLoggedIn){
+    if (isLoggedIn) {
       // If user is logged in, show loading & fetch their profile data
-      getUserProfile().then( ({ resource }) => {
+      getUserProfile().then(({ resource }) => {
         setFormValues({
           id: resource.id,
           user_name: resource.user_name,
+          profile_pic: resource.avatar,
+          business_name: resource.business_name,
+          specialty: resource.specialty,
+          tip_url: resource.tip_url,
+          video_url: resource.video_url,
+          blurb: resource.blurb,
         })
         setIsLoading(false)
-      }).catch( () => {
+      }).catch(() => {
         setIsLoading(false)
       })
     } else {
       // If user is logged out, kick them to the sign-in
-      router.push('/login')
+      router.push('/login').then(() => {
```

Comment:
The function you're passing to `.then` doesn't do anything, what's the purpose of this `.then` call?

Reply:
No purpose, removed.